test/specs/scenarios/crossDevice.js

```diff
@@ -291,5 +291,25 @@ export const crossDeviceScenarios = async (lang) => {
       crossDeviceSubmit.clickOnSubmitVerificationButton()
       verificationComplete.verifyUIElements(copy)
     })
+
+    it('should hide logo on all screens when hideOnfidoLogo is enabled and given token has feature enabled', async () => {
+      driver.get(`${baseUrl}&hideOnfidoLogo=true`)
+      welcome.continueToNextStep()
```

Comment:
We could also check for the logo on Welcome, DocumentSelector and Document Upload screens.

Reply:
Perfect, I was wondering if we could do something like this. Thank you, I'll make this change.